### DISCUSSION TITLE
is there anyway to change project server region?

### DISCUSSION THREAD

**Discussion by yamin8000** [supabase/supabase#2628] [Questions] [answered]
I'm still not sure if there's a restriction in my ISP that prevents me from connecting to my endpoints or some restrictions by amazon, BTW I'm in Iran so there's some laws that prevents USA companies to work with us. Anyway I realized if project region is `ap-south-1` there's no problem but for `eu-west-2` region sometimes I can't access my API. So I want to know is there any way to change region instead of creating a new project and importing all of the data?

**Comment by silentworks** [comment, marked as answer]
You can't change region after the database has been created. What you can do however is to create a new project, export the existing project's database and then importing it into the new project.

**Reply by yamin8000** [reply]
Maybe I'm doing something wrong but try exporting data from supabase dashboard to CSV and then import it when you're creating a new table, it doesn't allow it and says:
`Sorry! We only accept CSV, TSV, or XLSX file types, please upload another file.`

**Reply by lawrencecchen** [reply]
Had to do this recently. Following https://supabase.io/docs/guides/database#migrating-between-projects and using psql worked reasonably.

**Reply by yamin8000** [reply]
> 
> 
> Had to do this recently. Following https://supabase.io/docs/guides/database#migrating-between-projects and using psql worked reasonably.

But this... does put a smile on my face.
